content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -151,6 +151,16 @@ and also set a value for the `ImageMaximumGCAge` field in the kubelet configurat
 The value is specified as a Kubernetes _duration_; for example, you can set the configuration
 field to `3d12h`, which means 3 days and 12 hours.
 
+{{< note >}}
+```suggestion
+This feature does not track image usage across kubelet restarts. If the kubelet
+is restarted, the tracked image age is reset, causing the kubelet to wait the full
+`ImageMaximumGCAge` duration before qualifying images for garbage collection
+based on image age.
+```
+
+{{< /note>}}
```

Comment:
The content within the markdown block was [suggested (here)](https://github.com/kubernetes/website/pull/45153#discussion_r1528329516) for addition in the PR, it seems that the backticks (```) should not be included in the actual change. If that is the case, it would be appropriate to remove these backticks.

```suggestion
{{< note >}}
This feature does not track image usage across kubelet restarts. If the kubelet
is restarted, the tracked image age is reset, causing the kubelet to wait the full
`ImageMaximumGCAge` duration before qualifying images for garbage collection
based on image age.
{{< /note>}}
```